scheduler/scheduler-server/src/test/java/functionaltests/utils/SchedulerFunctionalTestNoRestart.java

```diff
@@ -40,6 +43,10 @@
 
     @BeforeClass
     public static void doNotStartSchedulerIfNotNeeded() throws Exception {
+
+        //Check if IAM microservice is required for authentication
```

Comment:
I think you could remove this comment at all the place and rename you method to express it, e.g. `prepareIAMIfNeeded()` or `checkWhetherIAMRequiredAndStart()` or something else.

Reply:
Done